prepare-build.sh

```diff
@@ -67,7 +67,27 @@ else
     if [ "$#" -gt 0 ]; then
         export MACHINE=${1}
     else
-        export MACHINE=dory
+        echo "***** No target defined *****\n"
+        echo "Usage:"
+        echo "$ . ./prepare-build.sh [update | device-name]\n"
```

Comment:
This is a good time to document what update does

Reply:
My understanding of the [] is that the argument is not required but here it is required